inference-engine/tests/ie_test_utils/functional_test_utils/src/ov_tensor_utils.cpp

```diff
@@ -52,6 +52,54 @@ ov::runtime::Tensor create_and_fill_tensor(
     return tensor;
 }
 
+ov::runtime::Tensor create_and_fill_tensor_unique_sequence(const ov::element::Type element_type,
+                                                           const ov::Shape& shape,
+                                                           const int32_t start_from,
+                                                           const int32_t resolution,
+                                                           const int seed) {
+    auto tensor = ov::runtime::Tensor{element_type, shape};
+    auto range = shape_size(shape) * 2;
+#define CASE(X)                                                                                           \
```

Comment:
Seems like we can extract the body into separate function and reuse with create_and_fill_tensor. 
Alternatively we can have filling mode as create_and_fill_tensor parameter. 
I would let @iefode to decide.

Reply:
So far we have many helpers `fill_data_random`/`fill_random_unique_sequence`/`fill_data_consistently`/`fill_data_random_float`/`fill_data_normal_random_float`/`fill_data_float_array`/`fill_data_roi`/`fill_data_const`/`fill_data_with_broadcast`...,  and they don't have same or even similar signature, so add a mode as parameter is not enough to make a unique API.

Reply:
Maybe we can keep this PR focused on Bucketize dynamic shape support and take the test-framework refinement as a separate task?

Reply:
@usstq for me sounds good, but still would let @iefode to make a decision